README.md

```diff
@@ -27,3 +27,35 @@
 <p align="center">
   <a href="https://www.open.mp">open.mp</a>
 </p>
+
+<hr>
+
+## Development
+
+### Requirements:
+
+- Git
+- Node.js
+- npm
+
+### Setup
+
+```bash
+# clone the project
+git clone https://github.com/openmultiplayer/wiki.git
+
+# Move into the directory
+cd wiki
+
+# Install the dependencies
+npm install
+
+# Start the server
+npm start
+```
+
+By default the development server will run on `http://localhost:3001` but can be changed with the following command:
```

Comment:
It's 3000 by default

Reply:
Oops, guess I have something running on 3000 already. Changed!